### PR TITLE
feat(issue-owners): Add to Code Owners and eval flow

### PR DESCRIPTION
**Pull request by NisanthanNanthakumar**
## Objective:
Explains the relationships between Ownership Rules, Code Owners and Code Mappings. I also added a section on the Evaluation Flow with an example and a section on how to setup Code Mappings, a prereq for Code Owners.

localhost_3000_product_issues_issue-owners_ (1)

### CODE REVIEW

File: src/docs/product/issues/issue-owners/index.mdx [hunk 1]
```diff
@@ -11,13 +11,13 @@ redirect_from:
...
 ---
 
-The Issue Owners feature allows you to create rules to decide which user or team should own an [issue](/product/data-management-settings/event-grouping/). These rules resemble a typical code owners file in a repository, and can match on file paths of files in the stack trace, URL of the request, or event tags. You can automatically assign issues to their respective owners, or alert the owners about the issue, allowing you to find the developer with the most context about a fix.
+Sentry offers multiple ways to define the "ownership" of an issue. With ownership defined, we can automatically assign issues and send alerts to the owner. Sentry defines ownership with Code Owners and Ownership Rules. Code Owners lets you import your CODEOWNERS file and we will assign according to those filepaths. Ownership Rules allows you to override the assignments from Code Owners and provide advanced matcher types (ex: urls, tags) and can match on file paths of files in the stack trace, URL of the request, or event tags.
```
Comment (AniketDas-Tekky) [inline src/docs/product/issues/issue-owners/index.mdx:14]: Can we link to the Codeowners documentation on github?

**Review by AniketDas-Tekky — COMMENTED** [review]
LGTM

**Review by imatwawana — CHANGES_REQUESTED** [review]
Overall, a really great addition! I've made several style fixes mostly around when and when not to capitalize things. If we're talking about the general concept of something, no caps (e.g., sometimes we're just talking an issue owner as a person), and if something isn't a high-level feature, or we're not referencing it directly in the UI, then also no caps (e.g., ownership rules, code owners). 

Also, you should make updates to these three files that reference ownership rules:
https://docs.sentry.io/product/sentry-basics/enrich-data/#issue-ownership
https://docs.sentry.io/product/issues/states-triage/#assign
https://docs.sentry.io/product/alerts/best-practices/#routing

File: src/docs/product/issues/issue-owners/index.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (NisanthanNanthakumar) [reply src/docs/product/issues/issue-owners/index.mdx:14]: @imatwawana the features are called Code Owners and Ownership Rules. I don't think it makes sense to have them lowercase.

[the remark this answers is not in the document]
Comment (imatwawana) [reply src/docs/product/issues/issue-owners/index.mdx:14]: We have _a lot_ of features so capitalizing all of them becomes cumbersome and when everything is capitalized, the importance of capitalization becomes diluted. Also, the capitalization across docs tends to be inconsistent, so I'm trying to impose a bit more structure in general around when we use it. As a style rule, we capitalize at the page level (so in this case Issue Owners page), but not features that are within a page unless we're doing a direct reference to them in the UI (in which case they'd be in quotation marks, e.g., the "Ownership Rules" section).

File: src/docs/product/issues/issue-owners/index.mdx [hunk 2]
```diff
@@ -11,13 +11,13 @@ redirect_from:
...
 ---
 
-The Issue Owners feature allows you to create rules to decide which user or team should own an [issue](/product/data-management-settings/event-grouping/). These rules resemble a typical code owners file in a repository, and can match on file paths of files in the stack trace, URL of the request, or event tags. You can automatically assign issues to their respective owners, or alert the owners about the issue, allowing you to find the developer with the most context about a fix.
+Sentry offers multiple ways to define the "ownership" of an issue. With ownership defined, we can automatically assign issues and send alerts to the owner. Sentry defines ownership with _code owners_ and _ownership rules_. Code owners lets you import your GitHub or GitLab CODEOWNERS file, and then we assign issues according to those file paths. Ownership rules allow you to override the assignments from code owners and provide advanced matcher types (for example, urls and tags). These rules can also match on the file paths of files in the stack trace, URL of the request, or event tags.
```
Comment (imatwawana) [inline src/docs/product/issues/issue-owners/index.mdx:14]: Fixing a couple grammar issue here that arise with _code owners_ being plural. 

```suggestion
Sentry offers multiple ways to define the "ownership" of an issue. With ownership defined, we can automatically assign issues and send alerts to the owner. Sentry defines ownership with _code owners_ and _ownership rules_. Code owners functionality lets you import your GitHub or GitLab CODEOWNERS file, and then we assign issues according to those file paths. Ownership rules allow you to override the assignments based on code owners and provide advanced matcher types (for example, urls and tags). These rules can also match on the file paths of files in the stack trace, URL of the request, or event tags.
```

**Review by imatwawana — APPROVED** [review]

File: src/docs/product/issues/issue-owners/index.mdx [hunk 3]
```diff
@@ -11,13 +11,13 @@ redirect_from:
...
 ---
 
-The Issue Owners feature allows you to create rules to decide which user or team should own an [issue](/product/data-management-settings/event-grouping/). These rules resemble a typical code owners file in a repository, and can match on file paths of files in the stack trace, URL of the request, or event tags. You can automatically assign issues to their respective owners, or alert the owners about the issue, allowing you to find the developer with the most context about a fix.
+Sentry offers multiple ways to define the "ownership" of an issue. With ownership defined, we can automatically assign issues and send alerts to the owner. Sentry defines ownership with _code owners_ and _ownership rules_. Code owners functionality lets you import your GitHub or GitLab CODEOWNERS file, and then we assign issues according to those file paths. Ownership rules allow you to override the assignments based on code owners and provide advanced matcher types (for example, urls and tags). These rules can also match on the file paths of files in the stack trace, URL of the request, or event tags.
```
Comment (imatwawana) [inline src/docs/product/issues/issue-owners/index.mdx:14]:
```suggestion
Sentry offers multiple ways to define the "ownership" of an issue. With ownership defined, we can automatically assign issues and send alerts to the owner. Sentry defines ownership with _code owners_ and _ownership rules_. Code owners functionality lets you import your [GitHub or GitLab CODEOWNERS file](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners), and then we assign issues according to those file paths. Ownership rules allow you to override the assignments based on code owners and provide advanced matcher types (for example, urls and tags). These rules can also match on the file paths of files in the stack trace, URL of the request, or event tags.
```

Comment (imatwawana) [inline src/docs/product/issues/issue-owners/index.mdx:14]:
```suggestion
Sentry offers multiple ways to define the "ownership" of an issue. With ownership defined, we can automatically assign issues and send alerts to the owner. Sentry defines ownership with _code owners_ and _ownership rules_. Code owners functionality lets you import your [GitHub](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) or [GitLab](https://docs.gitlab.com/ee/user/project/code_owners.html) CODEOWNERS file, and then we assign issues according to those file paths. Ownership rules allow you to override the assignments based on code owners and provide advanced matcher types (for example, urls and tags). These rules can also match on the file paths of files in the stack trace, URL of the request, or event tags.
```

File: src/docs/product/alerts/best-practices.mdx [hunk 4]
```diff
@@ -49,7 +49,7 @@ One way to keep alerts from becoming too noisy is to use filters, or “if” co
 
 These routing best practices ensure that you alert the right people about a problem in your application.
 
-- **Issue owners**: Use [issue owners](/product/issues/issue-owners/) to let Sentry automatically send alerts to the right people, as well as to ease configuration burden. You can configure ownership rules in **[Project] > Settings > Issue Owners**. When there are no matching owners, the alert goes to all project members by default. If this is too broad, and you'd like a specific owner to be the fallback, end your ownership rules with a rule like `*:<owner>`.
+- **Issue owners**: Use [ownership rules and code owners](/product/issues/issue-owners/) to let Sentry automatically send alerts to the right people, as well as to ease configuration burden. You can configure ownership rules in **[Project] > Settings > Issue Owners**. When there are no matching owners, the alert goes to all project members by default. If this is too broad, and you'd like a specific owner to be the fallback, end your ownership rules with a rule like `*:<owner>`.
```
Comment (imatwawana) [inline src/docs/product/alerts/best-practices.mdx:52]:
```suggestion
- **Issue owners**: Use [ownership rules and code owners](/product/issues/issue-owners/) to let Sentry automatically send alerts to the right people, as well as to ease configuration burden. You can configure ownership in **[Project] > Settings > Issue Owners**. In the case of ownership rules, when there are no matching owners, the alert goes to all project members by default. If this is too broad, and you'd like a specific owner to be the fallback, end your ownership rules with a rule like `*:<owner>`.
```

File: src/docs/product/sentry-basics/enrich-data/index.mdx [hunk 5]
```diff
@@ -71,13 +71,13 @@ Learn more in [Discover](/product/discover-queries/).
...
-
-This means that by adding meaningful **custom tags** to your errors, you'll be able to reduce noise by pinpointing notifications to the right people and automatically assigning new issues to their proper owners.
+Issue ownership allows you to target email notifications and automatically assign issues to members in your project teams, by defining ownership rules in your project settings or using code owners functionality. Ownership rules can be defined based on an error's origin `URL`, the `path` of the files in the error's stack-trace, or an error's `tag` value.
 
 ![Issue Owner Tag Rule](005.png)
 
-Learn more in [Issue Owners](/product/error-monitoring/issue-owners/).
+Code owners functionality lets you import your GitHub or GitLab CODEOWNERS file, and then we assign issues according to those file paths.
+
+You'll be able to reduce noise by pinpointing notifications to the right people and automatically assigning new issues to their proper owners. You can configure ownership rules in **[Project] > Settings > Issue Owners**. Learn more in [Issue Ownership](/product/error-monitoring/issue-owners/).
```
Comment (imatwawana) [inline src/docs/product/sentry-basics/enrich-data/index.mdx:80]:
```suggestion
You'll be able to reduce noise by pinpointing notifications to the right people and automatically assigning new issues to their proper owners. You can configure issue ownership in **[Project] > Settings > Issue Owners**. Learn more in [Issue Ownership](/product/error-monitoring/issue-owners/).
```

File: src/docs/product/issues/states-triage/index.mdx [hunk 6]
```diff
@@ -55,7 +55,7 @@ While you can work to have zero unresolved issues ("Inbox Zero") by resolving or
 
 ### Assign
 
-Assign issues to teams or individuals to route them to the right people and help you and your team see only relevant issues. Typically, assigning an issue to a team indicates that the team owns the issue and that they're responsible for triaging it further. You can assign issues manually or automatically using [ownership rules](/product/issues/issue-owners/). On the issue list, you might also see a list of suggested assignees including the owners of suspect commits.
+Assign issues to teams or individuals to route them to the right people and help you and your team see only relevant issues. Typically, assigning an issue to a team indicates that the team owns the issue and that they're responsible for triaging it further. You can assign issues manually or automatically using [issue ownership](/product/issues/issue-owners/). On the issue list, you might also see a list of suggested assignees including the owners of suspect commits.
```
Comment (NisanthanNanthakumar) [inline src/docs/product/issues/states-triage/index.mdx:58]:
```suggestion
Assign issues to teams or individuals to route them to the right people and help you and your team see only relevant issues. Typically, assigning an issue to a team indicates that the team owns the issue and that they're responsible for triaging it further. You can assign issues manually or automatically using [ownership rules and code owners](/product/issues/issue-owners/). On the issue list, you might also see a list of suggested assignees including the owners of suspect commits.
```

**Pull request merged**